### PR TITLE
add break change document

### PR DESCRIPTION
**Pull request by lengrongfu**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,11 @@ English version|[中文版](README_cn.md)
 
 **HAMi is a [Cloud Native Computing Foundation](https://cncf.io/) Landscape project.**
 
+## Break Change
+We change `device-plugin` env var name from `NodeName` to `NODE_NAME`, if you use the image version `v2.3.9`, you may encounter the situation that `device-plugin` cannot start, there are two ways to fix it:
+- Manually execute `kubectl edit daemonset` to modify the `device-plugin` env var from `NodeName` to `NODE_NAME`.
+- Upgrade to the latest version using helm, the latest version of `device-plugin` image version is `v2.3.10`, execute `helm upgrade hami hami/hami -n kube-system`, it will be fixed automatically.
+
```
Comment (wawa0210) [inline README.md]: I don’t think this should be placed so high in the readme. It is recommended to put it after `know issues`

Comment (lengrongfu) [reply README.md]: move to `know issues`.

**Comment by wawa0210** [comment]
thx

**Pull request merged**